correct heat ordering

Zipfer opened the pull request:
-fix file ordering
  -fix resource ordering
  -Bug #1250776 https://bugs.launchpad.net/fuel/+bug/1250776

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/649/

aglarendil commented:
+2

Zipfer committed:
correct heat ordering

  -fix file ordering
  -fix resource ordering
  -Bug #1250776 https://bugs.launchpad.net/fuel/+bug/1250776

xenolog commented on deployment/puppet/heat/manifests/engine.pp:
```diff
@@ -96,6 +96,7 @@
       $ocf_script_template = 'heat_engine_ubuntu.ocf.erb'
     }
 
+    Exec[create_ocf_dirs] ->
```
I can't find this resource in project. 
You tested it? 
how???
